<div class="profile-container">
    <div class="profile-photo">
        <img src="/images/people/LuCheng.webp" alt="Lu Cheng">
    </div>
    <div class="profile-info">
        <h1 class="profile-name">Lu Cheng</h1>
        <h2 class="profile-title">Principal Investigator</h2>
        <p class="profile-department">Department of Computer Science, Aalto University</p>
        <div class="profile-links">
            <a href="https://github.com/chengl7" class="profile-link">
                <i class="fab fa-github"></i> GitHub
            </a>
            <a href="https://www.linkedin.com/in/lu-cheng-7b502b258" class="profile-link">
                <i class="fab fa-linkedin"></i> LinkedIn
            </a>
            <a href="https://scholar.google.fi/citations?user=Zy14QK0AAAAJ&hl=en" class="profile-link">
                <i class="fas fa-graduation-cap"></i> Google Scholar
            </a>
            <a href="https://orcid.org/0000-0002-6391-2360" class="profile-link">
                <i class="fab fa-orcid"></i> ORCID
            </a>
        </div>
    </div>
</div>

I am the Principal Investigator (Academy Research Fellow) leading the Computational Genomics Lab research group at the Department of Computer Science, Aalto University. My main research interests are developing statistical and machine learning methods for applications in single cell biology, metagenomics, epigenetics.

### Research Interests
* Single cell bioinformatics
* Metagenomics
* Machine learning
* Bayesian statistics


## Education
- Ph.D. in Bayesian Statistics, Department of Mathematics and Statistics, University of Helsinki, 2013
- M.Sc., Department of Computer Science, University of Helsinki, 2009
- B.Sc., Department of Computer Science, Wuhan University, 2007

## Selected Publications

### Raw signal segmentation for estimating RNA modification from Nanopore direct RNA sequencing data

**Authors**: Cheng, Guangzhao, Vehtari, Aki, Cheng, Lu*

**Journal**: *eLife* 

**DOI**: [10.7554/elife.104618.1](https://doi.org/10.7554/elife.104618.1)

**Publisher**: eLife Sciences Publications, Ltd

---

### KMAP: Kmer Manifold Approximation and Projection for visualizing DNA sequences

**Authors**: Fu, Chengbo, Niskanen, Einari A, Wei, Gong-Hong, Yang, Zhirong, Sanvicente-García, Marta, Güell, Marc, Cheng, Lu*

**Journal**: *Genome Research(accepted, to appear)* 

**DOI**: [10.1101/2024.04.12.589197](https://doi.org/10.1101/2024.04.12.589197)

**Publisher**: Cold Spring Harbor Laboratory

---

### NanoBaseLib: A Multi-Task Benchmark Dataset for Nanopore Sequencing

**Authors**: Cheng, Guangzhao, Fu, Chengbo, Cheng, Lu*

**Journal**: *Advances in Neural Information Processing Systems (NeurIPS)* 37

**Link**: [NeurIPS Proceedings](https://papers.nips.cc/paper_files/paper/2024/hash/8bce223b376f52fb86a148097eebb10d-Abstract-Datasets_and_Benchmarks_Track.html)

**Publisher**: Curran Associates, Inc.

---

### SCAPE: a mixture model revealing single-cell polyadenylation diversity and cellular dynamics during cell differentiation and reprogramming

**Authors**: R. Zhou, X. Xiao, P. He, Y. Zhao, M. Xu, X. Zheng, R. Yang, S. Chen, L. Zhou, D. Zhang, Q. Yang, J. Song, C. Tang, Y. Zhang, J.-W. Lin*, L. Cheng* and L. Chen*, 

**Journal**: *Nucleic Acids Research*

**Link**: [10.1093/nar/gkac167](https://doi.org/10.1093/nar/gkac167)

---

### An additive Gaussian process regression model for interpretable non-parametric analysis of longitudinal data

**Authors**: L. Cheng*, S. Ramchandran, T. Vatanen, N. Lietzén, R. Lahesmaa, A. Vehtari and H. Lähdesmäki*

**Journal**: *Nature Communications*

**Link**: [10.1038/s41467-019-09785-8](https://doi.org/10.1038/s41467-019-09785-8)

---

Full publication list in [Google Scholar]([10.1038/s41467-019-09785-8](https://doi.org/10.1038/s41467-019-09785-8))

---


## Current Projects
- LLM for scRNA-seq data
- RNA modification estimation using Nanopore data
- motif discovery by k-mer manifold learning

## Teaching
- Bayesian model applications in biomedicine (Spring 2025, UEF)
- Biological data visualization (Spring 2023, Aalto)
- Computational Genomics (Autumn 2021, 2022, 2023, Aalto)

## Professional Activities
- Chair of Biostatics track, 19th conference on Computational Intelligence methods for Bioinformatics and Biostatistics (CIBB24)
- Reviewer for NeurIPS (2016-2024), ICLR (2018-2024), ICML (2025)
- Reviewer for 
  + Nature Microbiology
  + IEEE Transactions on Signal Processing
  + Neurocomputing
  + Ecology Evolution
  + Genome Biology Evolution
  + Plos Computational Biology
  + Plos One
  + BMC Bioinformatics
  + Nucleic Acid Research
  + Microbiome
  + Microbiology Spectrum

---
*Last updated: March 2025*

[Back to Lab Members](/people)  
 
 
